app/models/link.rb

```diff
@@ -1,4 +1,5 @@
 class Link < ApplicationRecord
   belongs_to :service
   validates_presence_of :label, :url
+  validates_uniqueness_of :label, scope: :service
```

Comment:
Ooh good point, I hadn't thought to check last week. Some of them look like dupes and some are not, although they definitely could do with renaming because they're well confusing (see attached screenshot 🤯)

There aren't too many services where this has happened. I will check to see if this validation means that we need to update the links before any other changes to those services can be made.

<img width="649" alt="Screenshot 2022-08-08 at 13 48 51" src="https://user-images.githubusercontent.com/11888656/183425019-0056e780-06b9-4bd5-9d1b-168e1b48344d.png">